txpipe/sysmaps.py

```diff
@@ -142,12 +143,16 @@ def run(self):
                 shear_tmp = {'mcal_g1': shear_data['true_g1'], 'mcal_g2': shear_data['true_g2']}
             else:
                 shear_tmp = {'mcal_g1': shear_data['mcal_g1'], 'mcal_g2': shear_data['mcal_g2']}
+                shear_psf_tmp = {'mcal_psf_g1': shear_data['mcal_psf_g1'], 'mcal_psf_g2': shear_data['mcal_psf_g2']}
```

Comment:
in this line I think you need to call the dict keys just 'mcal_g1' and 'mcal_g2', because that's what the mapper is expecting (actually maybe better to modify the mapper in mapping/basic_maps.py to just look for g1 and g2 as keys and use those - cleaner)